src/NuGet.Clients/NuGet.VisualStudio.Common/ServiceLocator.cs

```diff
@@ -118,7 +117,7 @@ public static void InitializePackageServiceProvider(IAsyncServiceProvider provid
                 }
             }
 
-            return Package.GetGlobalService(typeof(TService)) as TInterface;
```

Comment:
Dead code. This method requires the UI thread, which defeats the purpose of this method which indicates that it's free threaded.